src/categories/create.js

```diff
@@ -48,7 +48,7 @@ module.exports = function(Categories) {
 			function(data, next) {
 				category = data.category;
 
-				var defaultPrivileges = ['find', 'read', 'topics:read', 'topics:create', 'topics:reply', 'upload:post:image'];
+				var defaultPrivileges = ['find', 'read', 'topics:read', 'topics:create', 'topics:reply', 'posts:edit', 'posts:delete', 'upload:post:image'];
```

Comment:
I noticed the upgrade script gives `topics:delete` privilege to existing categories if a group or user hash `topics:create`, but new categories don't get this privilege.